src/MSBuildLocator/VisualStudioInstanceQueryOptions.cs

```diff
@@ -29,6 +29,16 @@ public class VisualStudioInstanceQueryOptions
         /// </summary>
         public DiscoveryType DiscoveryTypes { get; set; }
 
+#if NETCOREAPP
+        /// <summary>
+        ///     Allow discovery of .NET SDK versions that are unlikely to be successfully loaded in the current process.
+        /// </summary>
+        /// <remarks>
+        ///     Defaults to <see langword="false"/>. Set this to <see langword="true"/> only if your application has special logic to handle loading an incompatible SDK, such as launching a new process with the target SDK's runtime.
+        /// </remarks.
+        public bool AllowQueryAllRuntimeVersions { get; set; } = false;
```

Comment:
```suggestion
        public bool AllowAllRuntimeVersions { get; set; } = false;
```

I admit the 'query' made it hard to read for me, but you can ignore this. (Query is implied in the type name anyways.)